graphics/vtk/Portfile

```diff
@@ -122,6 +127,7 @@ foreach pyver ${python_versions} {
             }
     }
 
+
```

Comment:
There's several additions of extraneous space. It's just cosmetic, but I would revert them.

Reply:
I added a commit to remove the extra whitespace. It can be squashed into the first commit when merged.